apps/explorer/src/utils/searchUtil.ts

```diff
@@ -11,15 +9,8 @@ const deduplicate = (results: string[] | undefined) =>
 let navigateWithUnknown: Function;
 let overrideTypeChecks = false;
 
-if (IS_STATIC_ENV) {
-    import('./static/searchUtil').then((uf) => {
-        navigateWithUnknown = uf.navigateWithUnknown;
-        overrideTypeChecks = true;
-    });
-} else {
-    import('./api/searchUtil').then(
-        (uf) => (navigateWithUnknown = uf.navigateWithUnknown)
-    );
-}
+import('./api/searchUtil').then(
```

Comment:
This can just statically be imported now, no need to use dynamic imports with this reassignment nonsense.